app/services/csv_handler.rb

```diff
@@ -8,82 +8,129 @@ def upload(file)
   end
 
   def parse_room_csv(schedule_id)
-    begin
-      ActiveRecord::Base.transaction do
-        room_data = CSV.parse(@file.read, headers: true)
-        room_data.each do |row|
-          Room.create!(
-            schedule_id: schedule_id,
-            campus: row['campus'],
-            building_code: row['building_code'],
-            room_number: row['room_number'],
-            capacity: row['capacity'].to_i,
-            is_lecture_hall: row['is_lecture_hall'] == 'True',
-            is_learning_studio: row['is_learning_studio'] == 'True',
-            is_lab: row['is_lab'] == 'True',
-            is_active: row['is_active'] == 'True',
-            comments: row['comments']
-          )
-        end
+    ActiveRecord::Base.transaction do
+      room_data = CSV.parse(@file.read, headers: true)
+      room_data.each do |row|
+        Room.create!(
+          schedule_id: schedule_id,
+          campus: row['campus'],
+          building_code: row['building_code'],
+          room_number: row['room_number'],
+          capacity: row['capacity'].to_i,
+          is_lecture_hall: row['is_lecture_hall'] == 'True',
+          is_learning_studio: row['is_learning_studio'] == 'True',
+          is_lab: row['is_lab'] == 'True',
+          is_active: row['is_active'] == 'True',
+          comments: row['comments']
+        )
       end
-      # Flash data to be received by controller
-      return { notice: 'Rooms successfully uploaded.' }
-    rescue StandardError => e
-      return { alert: "There was an error uploading the CSV file: #{e.message}" }
     end
+    # Flash data to be received by controller
+    { notice: 'Rooms successfully uploaded.' }
+  rescue StandardError => e
+    { alert: "There was an error uploading the CSV file: #{e.message}" }
   end
 
   def parse_instructor_csv(schedule_id)
-    begin
```

Comment:
Why was begin removed?

Reply:
I ran rubocop -a that might be why, I never took that line on purpose